lib/Target/AVR/AVRTargetMachine.cpp

```diff
@@ -25,7 +25,7 @@
 namespace llvm {
 
 /// Processes a CPU name.
-static StringRef getTargetCPU(StringRef CPU) {
+static StringRef GetTargetCPU(StringRef CPU) {
```

Comment:
> clashes with a function

Yup, that's why I went with the capital letter (and because one of the calls right below used a capital letter). Can rename!